edx_name_affirmation/tasks.py

```diff
@@ -43,7 +43,7 @@ def idv_update_verified_name_task(self, attempt_id, user_id, name_affirmation_st
     # want to grab all verified names for the same user and name combination, because
     # some of those records may already be associated with a different IDV attempt.
     verified_names = VerifiedName.objects.filter(
-        (Q(verification_attempt_id=attempt_id) | Q(verification_attempt_id__isnull=True))
+        (Q(platform_verification_attempt_id=attempt_id) | Q(platform_verification_attempt_id__isnull=True))
```

Comment:
both here and with delete this library now assumes any future idv change is related to the new model, we've completely dropped support for `SoftwareSecurePhotoVerification` changes.